change 3rd parameter of circle to be diameter instead of radius

montoyamoraga opened the pull request:
This pull request addresses #3494
I changed both the definition and the documentation of the circle() function,
so that its third parameter by default is diameter, instead of radius.

hellonearthis commented:
I think it should default to radius and use ellipseMode to change the default to diameter.

This pull would break too much.

meiamsome commented on src/core/shape/2d_primitives.js:
```diff
@@ -230,9 +230,9 @@ p5.prototype.ellipse = function(x, y, w, h, detailX) {
-  var args = Array.prototype.slice.call(arguments, 0, 2);
-  args.push(2 * arguments[2]);
-  args.push(2 * arguments[2]);
+  let args = Array.prototype.slice.call(arguments, 0, 2);
```
`let` is not yet permitted in the source of p5, please use `var` for now

montoyamoraga replied:
thanks! just fixed it.

intuitivetextmining commented:
default should be diameter

intuitivetextmining commented:
I've made changes which 

- work and are tested with sketches
- also update the documentation

[https://github.com/processing/p5.js/issues/3494#issuecomment-460859136](https://github.com/processing/p5.js/issues/3494#issuecomment-460859136)

montoyamoraga commented:
> I've made changes which
> 
> * work and are tested with sketches
> * also update the documentation
> 
> [#3494 (comment)](https://github.com/processing/p5.js/issues/3494#issuecomment-460859136)

hi! sorry i didn't notice your comment, I think we wrote the same thing?
I just went to the file and changed everything to diameter, and deleted the 2* part.
I think this pull request matches your proposed change, right?

makeyourownalgorithmicart commented:
thanks @montoyamoraga - let's hope your PR goes through and the next release is soon!

makeyourownalgorithmicart commented:
thanks everyone!

do we know when the next release is? (and will this be in it?)

I'm asking anyone to rush - but if the timeframe is long then I'll put circle back into my simple.js as I use it for teaching ...

lmccart commented:
thanks @montoyamoraga. @makeyourownalgorithmicart this will be in the next release. there is no scheduled date as of right now.
